pig/src/main/java/com/netflix/iceberg/pig/IcebergPigInputFormat.java

```diff
@@ -36,7 +36,7 @@
 import com.netflix.iceberg.types.Type;
 import com.netflix.iceberg.types.TypeUtil;
 import com.netflix.iceberg.types.Types;
-import org.apache.commons.lang.SerializationUtils;
+import org.apache.commons.lang3.SerializationUtils;
```

Comment:
Did we need to use `commons-lang3`? Not entirely opposed to it, just wondering if we can minimize the diff just a tiny bit.

Reply:
Hmm the original import doesn't seem to be part of the natural dependency tree anymore :/

Reply:
That's fine then, actually we probably want to be exclusively using `lang3` in upstream as well. We'll catch this if we introduce Baseline and linting has a rule for it.